files/__main__.py

```diff
@@ -32,6 +32,7 @@
 
 if environ.get("SITE_ID") is None:
 	from dotenv import load_dotenv
+	load_dotenv(dotenv_path=Path("bootstrap/site_env"))
 	load_dotenv(dotenv_path=Path("env"))
```

Comment:
is this line supposed to still be here?

Reply:
Yes. `load_dotenv` is additive, and we'd expect users to modify their "env" file as a patch of local settings while leaving bootstrap/site_env as the defaults. It also seems to fail gracefully if a file isn't present. I'll add `override=True` to the second call, though, since the docs suggest that might matter for patching behavior.